detection_rules/kbwrap.py

```diff
@@ -150,6 +150,16 @@ def parse_list_id(s: str):
         click.echo(f' - {rule_str}')
     if response['errors']:
         handle_response_errors(response)
+    else:
+        def _process_imported_items(imported_items_list, item_type_description, item_key):
```

Comment:
```suggestion
        def _process_imported_items(imported_items_list, item_type_description, item_key):
            """Displays a success message, and lists their IDs."""
```

Or some alternative doc string.

Reply:
I agree, updated :+1: